untie/src/main.cpp

```diff
@@ -2,7 +2,18 @@
 
 using namespace knot;
 
-int main() {
+int main(int argc, char** argv) {
+    if (argc < 2) {
+        std::cout << "Usage:\n\tknoting [client/server]\n";
+    }
+    std::string arg = argv[1];
+    if (arg == "server") {
+        Engine::isClient = false;
+    } else if (arg != "client") {
+        std::cout << "Invalid Argument \n";
+        return 0;
+    }
```

Comment:
Check an else if other argument has been passed, and show an error

Reply:
This is what I am doing here, might not  be clear as I'm using it to check that the argument is also not client but it removes the need for an extra if statement